heartbeat/ocf.py

```diff
@@ -248,16 +248,16 @@ def __str__(self):
 		return self.to_xml()
 
 	def to_xml(self):
-		def opt(s, name, var):
+		def opt(s, name, var, time=True):
 			if var is not None:
 				if type(var) == int:
-					var = "{}s".format(var)
+					var = "{}{}".format(var, "s" if time else "")
 				return s + ' {}="{}"'.format(name, var)
 			return s
 		ret = '<action name="{}"'.format(self.name)
 		ret = opt(ret, "timeout", self.timeout)
 		ret = opt(ret, "interval", self.interval)
-		ret = opt(ret, "depth", self.depth)
+		ret = opt(ret, "depth", self.depth, time=False)
```

Comment:
Shouldnt these be += ?

Reply:
no, it takes the previous version of the string as the first parameter and returns the updated version.. so the += is done inside the opt function, if that makes sense.

Reply:
Ah. I didnt notice that :)